XML/TestCases/LIS-Tests.xml

```diff
@@ -1,15 +1,30 @@
-﻿<TestCases>
+<TestCases>
     <test>
         <testName>LIS-DRIVER-VERSION-CHECK</testName>
         <testScript>LIS-VERSION-CHECK.sh</testScript>
         <files>.\TestScripts\Linux\LIS-VERSION-CHECK.sh,.\TestScripts\Linux\utils.sh</files>
         <setupType>OneVM</setupType>
         <Platform>Azure</Platform>
         <Category>Functional</Category>
-        <Area>LIS</Area>
+        <Area>LIS_DEPLOY</Area>
```

Comment:
@chvalean this introduces new AREA. Any reason beyond 'LIS'?

Reply:
> @chvalean this introduces new AREA. Any reason beyond 'LIS'?

LIS_DEPLOY already exists for the rpm install scenarios.
We actually get rid of one area - the one called LIS.

Reply:
acked. From now on, there should be no 'LIS' in Area. Can you create an issue for 13 'LIS' Area in repo?